app/controllers/book_suggestions_controller.rb

```diff
@@ -1,4 +1,6 @@
 class BookSuggestionsController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
:thinking: if we use this skip validation, all methods in this controller skip authenticate action. It's ok, until now we only have create method, but it's a good practice to specify skipped methods

```suggestion
  skip_before_action :authenticate_user!, only: :create
```